### PR TITLE
Resolve PHP7 compatibility issues

### PR DESCRIPTION
**Pull request by emirb**
Fixes https://github.com/Mangopay/mangopay2-php-sdk/issues/165

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 
 language: php
 
 php:
   - 5.4
-
-before_install:
-  - sudo apt-get -y install git zip php5-cli php5-curl
+  - 7.0
+  - 7.1
```
Comment (iammichiel) [inline .travis.yml]: You might as well add the 5.5 and 5.6 version.

**Comment by emirb** [comment]
Do you know what's with the Sandbox API? `test_GetEventList_PayinNormalCreated` is always failing with Internal server error @iammichiel

**Comment by hobailey** [comment]
@emirb it's most likely related to https://github.com/Mangopay/mangopay/issues/1 I'm afraid.

**Comment by emirb** [comment]
@hobailey Thanks, then this PR can be merged, but note that the tests will be failing until performance improvements end those 503s.

**Comment by iammichiel** [comment]
I understand the objective to test against a real environment. But if the tests are considered unreliable, why even bother to write or maintain them? IMO the server-part should be mocked.

**Pull request merged**